CharlieBackend.Api/Validators/AccountDTOValidators/ChangeCurrentPasswordDtoValidator.cs

```diff
@@ -16,11 +13,15 @@ public ChangeCurrentPasswordDtoValidator()
                 .EmailAddress().WithMessage("Incorrect email")
                 .MaximumLength(50).WithMessage("Email cannot be greateh than {MaxLength} symbols");
             RuleFor(x => x.CurrentPassword)
-                .NotEmpty().WithMessage("{PropertyName} is required")
-                .MaximumLength(30).WithMessage("Password cannot be greater than {MaxLength} symbols");
+                .NotEmpty()
+                .MinimumLength(8)
+                .MaximumLength(30)
```

Comment:
Please, remove the recurring magic numbers

Reply:
Ok